### PR TITLE
Fix meta stables

### PR DESCRIPTION
**Pull request by fdcroce**
Fix typo error in method njoy.process_neutrons() and implementation of a new test in endf6.get_ace() to check it works correctly.
Edit worker endf6.endf_pendf_worker to take the correct ZA to emply in the name of ace file. generated The previous version was not able to name correctly in case of metastable nuclides.

### CODE REVIEW

File: sandy/core/endf6.py [hunk 1]
```diff
@@ -2219,12 +2224,15 @@ def endf6_perturb_worker(e6, pendf, n,
     intro = endf6_pert.read_section(mat, 1, 451)
     za = int(intro["ZA"])
     meta = int(intro["LISO"])
-    zam = sandy.zam.za2zam(za, meta=meta)
+    zam = sandy.zam.za2zam(za, meta=meta,method="")
```
Comment (luca-fiorito-11) [inline sandy/core/endf6.py]: This change is not needed

File: sandy/core/endf6.py [hunk 2]
```diff
@@ -2219,12 +2224,15 @@ def endf6_perturb_worker(e6, pendf, n,
...
+    zaid = ace_kws.get("zaid", "nndc")
+    if zaid == "nndc":
+        za = sandy.zam.zam2za(zam, method=zaid)[0]
     params = dict(
         MAT=mat,
         ZAM=zam,
-        ZA=za,
         META=meta,
+        ZA = za,
```
Comment (luca-fiorito-11) [inline sandy/core/endf6.py]: No spaces

**Review by luca-fiorito-11 — CHANGES_REQUESTED** [review]
Two minor comments

**Comment by luca-fiorito-11** [comment]
Can you also comment out the failing test? We'll address it later

File: sandy/core/endf6.py (hunk 1 above)
Comment (fdcroce) [reply sandy/core/endf6.py]: It is. The za retrieved the first part of endf6 file doesn't follow _"nndc"_, then if method za2zam is _"nndc"_ as default  meta information is discarded. This brings to generation of wrong zam.
Maybe better to change notation in _method=False_ consistently with examples in zam module.

Comment (luca-fiorito-11) [reply sandy/core/endf6.py]: ok, but add a space between "," and "method"

**Review by luca-fiorito-11 — APPROVED** [review]

**Review by luca-fiorito-11 — APPROVED** [review]

**Pull request merged**